knowledge-base/shivammathur/cache-extensions/actions-security.yml

```diff
@@ -0,0 +1,27 @@
+name: 'cache-extensions'
+outbound-endpoints:
+  api.github.com:443:
+    port: 443
+    reason: to call GitHub API
+  github.com:443:
+    port: 443
+    reason: to download scripts from GitHub
+  objects.githubusercontent.com:443:
+    port: 443
+    reason: to fetch git object
+  ppa.launchpad.net:80:
```

Comment:
The port needs to be specified in the `port` key. This applies to all entries. I will improve experience of creating this action-security.yml file directly from the harden-runner-link.

Reply:
Fixed